markdown/dev/reference/api/path/attr/en.md

```diff
@@ -5,16 +5,16 @@ title: Path.attr()
 This `Path.attr()` method can be used to add attributes to the Path object.
 It calls `this.attributes.add()` under the hood, and returns the Path object.
 
-If the third parameter is set to `true` it will call `this.attributes.set()`
+If the optional third parameter is set to `true` it will call `this.attributes.set()`
 instead, thereby overwriting the value of the attribute.
 
 ## Signature
 
 ```js
 Path path.attr(
   string name,
-  mixed value,
-  bool overwrite = false
+  mixed value[,
```

Comment:
Same remark. No need for the square brackets as the optional value has a default assigned.